lib/internal/Magento/Framework/Phrase/registration.php

```diff
@@ -0,0 +1,4 @@
+<?php
+if (!function_exists('__')) {
+    require '__function.php';
```

Comment:
What is the use for such registration file?

Reply:
@orlangur we could mark this file deprecated as well as it should get removed at the same time as app/functions.php

Reply:
Such file is a bit confusing, please just include `__.php` in `registration.php` of Magento Framework.